scripts/swagger.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+#http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#
+#Copyright 2017 Intel Corporation
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+swagger generate spec -o ${__dir}/../swagger.json
+
+perl -i -p -e 's/ProcessNodes/Process/g' ${__dir}/../swagger.json
```

Comment:
@mkleina, otherwise generated client does not work with Snap server as the data model is different. Unless we rename our variables inside Snap code base. I don't think @jcooklin let me do that.